pkg/vpcmodel/explainabilityInput.go

```diff
@@ -327,12 +327,12 @@ func (c *VPCConfig) getNodesFromAddress(ipOrCidr string, inputIPBlock *ipblock.I
 		if !inputIPBlock.ContainedIn(vpcAP) {
```

Comment:
`vpcAP` is computed from `c.VPC.AddressRange()` instead of `c.VPC.AddressPrefixes()`

```
func (v *VPC) AddressRange() *ipblock.IPBlock {
	return v.internalAddressRange
}
```

// internalAddressRange is the union of all the vpc's subnets' CIDRs

Reply:
Right. Why does `c.VPC.AddressPrefixes()` returns a string?
Can an address prefix be more than one `CIDR`? I believe it can.

Reply:
It returns a slice of strings:
```
func (v *VPC) AddressPrefixes() []string {
	return v.addressPrefixes
}
```

you can use ipblock package to get IPBlock from list of CIDR strings

Reply:
this is for a separate issue, to consider this change

Reply:
#505

Reply:
Turns out there are config objects that are missing the input address prefixes. ROI of having a WA for this not worth it. We'll stay with the subnets range